<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StacieEmu</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TTC1:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TTC2:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Console" name="ttc2Console"/>
      </item>
      <item row="1" column="0">
       <widget class="Console" name="ttc1Console"/>
      </item>
      <item row="1" column="2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="QComboBox" name="actionBox">
          <item>
           <property name="text">
            <string>ACK</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EXEC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>NACK</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Action:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Filename</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QComboBox" name="quickBox">
          <item>
           <property name="text">
            <string>STATUS TEST</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Quick Commands:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Command:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cmdBox">
          <item>
           <property name="text">
            <string>TRANSMIT</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RECEIVE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OPMODE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GETTELEMETRY</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GETSTATUS (debug)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Custom Commands:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Data File:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Select File</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QPushButton" name="quickButton">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>GPS:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Console" name="gpsConsole"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="stacieemu.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;onnect</string>
   </property>
   <property name="toolTip">
    <string>Connect to the serial ports</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="stacieemu.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from serial ports</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="stacieemu.qrc">
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="stacieemu.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Console</class>
   <extends>QPlainTextEdit</extends>
   <header>console.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="stacieemu.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionBeenden</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>about()</slot>
 </slots>
</ui>
